gpii/node_modules/preferencesServer/src/preferencesPutHandler.js

```diff
@@ -33,7 +33,7 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
             onGetRawPreferences: {
                 funcName: "gpii.preferencesServer.put.onGetRawPreferences",
                 args: [ "{preferencesServer}", "{preferencesServer}.ontologyHandler",
-                     "{request}", "{arguments}.0", "{that}.events.onPutRawPreferences" ]
+                     "{request}", "{requestProxy}", "{arguments}.0", "{that}.events.onPutRawPreferences" ]
```

Comment:
Don't inject the requestProxy here - it doesn't do anything that the request doesn't